plugins/citadel-paper/src/main/java/vg/civcraft/mc/citadel/listener/EntityListener.java

```diff
@@ -1,30 +1,23 @@
 package vg.civcraft.mc.citadel.listener;
 
-import java.util.ArrayList;
-import java.util.Iterator;
-import java.util.List;
-import java.util.UUID;
+import java.util.*;
```

Comment:
Please don't do wildcard imports

Reply:
Thanks and apologies. Was probably an auto format.